Wireframe/index.css

```diff
@@ -0,0 +1,52 @@
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+  } 
+
+ header {
+   background-color: #F1F1F1;
+   text-align: center;
+   padding: 20px;
+  } 
+ 
+ #logo {
+    display: block;
+    margin: 0 auto;
+ }
+ button{
+  padding: 10px, 10px;
+  background-color: #F1F1F1;
+  border-radius: 5px;
+
+ }
+
+  article:first-child {
+    background-color: brown;
+    border: 2px solid rgb(15, 15, 15);
+    padding: 20px 20px;
+    margin-bottom: 20px;
+  }
+
+  article:not(:first-child){
+    display: inline-block; 
+    border: 1px solid black;
+    background-color: brown;
+    width: 46%;
+    padding: 20px 20px;
+    margin-top: 0px;
+    vertical-align: top;
+    height: 250px;
+  }
+
+footer{
```

Comment:
This footer doesn't looked fixed to the bottom of the viewport. Try exploring position:fixed - it has some interesting effects.

https://css-tricks.com/absolute-relative-fixed-positioining-how-do-they-differ/